roles/ceph-docker-common/tasks/misc/ntp_redhat.yml

```diff
@@ -1,9 +1,4 @@
 ---
-- name: include ../checks/check_ntp_redhat.yml
```

Comment:
it can't start the service anymore since the package installation is in this include